app/exec/extension/_lib/extension-composer-factory.ts

```diff
@@ -21,6 +21,7 @@ export class ComposerFactory {
 			switch (target.id) {
 				case "Microsoft.VisualStudio.Services" :
 				case "Microsoft.VisualStudio.Services.Cloud" :
+				case "Microsoft.VisualStudio.Services.Resource.Cloud" :
```

Comment:
I'd like to know more about this installation target. Do we really want to treat it the same as a standard extension? Each target is responsible for performing its own validation, and since there are special validation steps for this Resource.Cloud target, it seems like it should have its own ExtensionComposer.